app/frontend/src/submissions/components/SubmissionForm/Location.vue

```diff
@@ -38,10 +38,23 @@ Licensed under the Apache License, Version 2.0 (the "License");
             id="wellStreetAddress"
             type="text"
             label="Street address"
+            @input="fetchAddressSuggestions"
+            v-on:focus="showList"
+            v-on:blur="hideList"
             :errors="errors['street_address']"
             :loaded="fieldsLoaded['street_address']"
             :disabled="sameAsOwnerAddress"
           ></form-input>
+           <!-- Display the address suggestions -->
+        <div v-if="addressSuggestions.length > 0" class="address-suggestions list-group list-group-flush border" id="address-suggestions-list">
+          <li v-for="(suggestion, index) in addressSuggestions" :key="index">
+            <button @mousedown="selectAddressSuggestion(suggestion)" class="list-group-item list-group-item-action border-0">{{ suggestion }}</button>
+          </li>
+        </div>
+        <!-- Display a loading indicator while fetching suggestions -->
```

Comment:
Consider adding some custom CSS to override the default behaviour of the boostrap class so it sits above the rest of the elements instead of pushing them down. Just needs some simple overrides, but don't use a z-index as high as I did :)
```css
specificity-of-elements-to-override-nicely {
  position: absolute;
  z-index: 1000000;
}
```

Example:
![image](https://github.com/bcgov/gwells/assets/62873746/407b5cbc-0c68-441c-ae09-cc6dbb877ec3)

Current (Spasms as lines are regenerated each request):
![image](https://github.com/bcgov/gwells/assets/62873746/07c97fff-d9e7-4e5e-ade1-6f4a79aebbfb)

Reply:
implemented